zerver/tests/test_events.py

```diff
@@ -1281,7 +1281,9 @@ def test_change_notification_sound(self) -> None:
         notification_setting = "notification_sound"
 
         events = self.verify_action(
-            lambda: do_change_notification_settings(self.user_profile, notification_setting, "ding")
+            lambda: do_change_notification_settings(
+                self.user_profile, notification_setting, "ding", acting_user=self.user_profile
```

Comment:
I'm passing `acting_user=self.user_profile` here for the test to be consistent with the surrounding code .

Reply:
sgtm